src/sentry/static/sentry/app/views/organizationDashboard/exploreWidget.jsx

```diff
@@ -81,42 +75,96 @@ class ExploreWidget extends React.Component {
-              <Button borderless size="zero" to={this.getExportToDiscover(query)}>
-                <InlineSvg src="icon-discover" />
-              </Button>
-              <Button borderless size="zero" to={this.getExportToEvents(query)}>
-                <InlineSvg src="icon-stack" />
-              </Button>
-            </ExploreActions>
-          </ExploreRow>
-        ))}
-      </DropdownLink>
+      <DropdownMenu>
+        {({isOpen, getRootProps, getActorProps, getMenuProps}) => {
+          return (
+            <ExploreRoot {...getRootProps()}>
+              <div {...getActorProps()}>
+                <ExploreButton isOpen={isOpen}>
+                  {t('Explore Data')}
+                  <Chevron isOpen={isOpen} src="icon-chevron-right" />
+                </ExploreButton>
+              </div>
+              <ExploreMenu {...getMenuProps({isStyled: true, isOpen})}>
+                {discoverQueries.map(query => (
+                  <ExploreRow key={query.name}>
+                    <QueryName>{query.name}</QueryName>
+
+                    <ExploreAction to={this.getExportToDiscover(query)}>
+                      <InlineSvg src="icon-discover" />
+                    </ExploreAction>
+                    <ExploreAction to={this.getExportToEvents(query)}>
+                      <InlineSvg src="icon-stack" />
+                    </ExploreAction>
+                  </ExploreRow>
+                ))}
+              </ExploreMenu>
+            </ExploreRoot>
+          );
+        }}
+      </DropdownMenu>
     );
   }
 }
 export default withOrganization(ExploreWidget);
 
+const ExploreRoot = styled('div')`
+  border-left: 1px solid ${p => p.theme.borderLight};
+  position: relative;
+`;
+
+const UnstyledButton = props => <Button borderless size="zero" {...props} />;
+
+const ExploreButton = styled(({isOpen, ...props}) => <UnstyledButton {...props} />)`
+  position: relative;
+  color: ${p => (p.isOpen ? p.theme.purple : p.theme.gray2)};
+  padding: ${space(1)} ${space(2)};
+  border-radius: 0 0 ${p => p.theme.borderRadius} 0;
+  ${p => p.isOpen && `z-index: ${p.theme.zIndex.dropdownAutocomplete.actor}`};
+  ${p => p.isOpen && `box-shadow: ${p.theme.dropShadowHeavy}`};
+  &:hover {
+    color: ${p => p.theme.purple};
+  }
+  transition: box-shadow 0.25s;
+`;
+
+const ExploreMenu = styled('div')`
+  display: ${p => (p.isOpen ? 'flex' : 'none')};
+  flex-direction: column;
+
+  position: absolute;
+  right: -1px;
+  bottom: calc(100% - 1px);
+  z-index: ${p => p.theme.zIndex.dropdownAutocomplete.menu};
+
+  background-color: white;
+  border: 1px solid ${p => p.theme.borderLight};
+  box-shadow: ${p => p.theme.dropShadowHeavy};
+`;
+
 const ExploreRow = styled('li')`
   display: flex;
-  justify-content: space-between;
   align-items: center;
   padding: 0 ${space(0.5)};
 `;
 
-const ExploreActions = styled('div')`
-  display: flex;
+const ExploreAction = styled(UnstyledButton)`
+  padding: ${space(1)} ${space(1)};
```

Comment:
```suggestion
  padding: ${space(1)};
```

You don't need duplicate padding values.